en/02_Developer_Guides/00_Model/11_Scaffolding.md

```diff
@@ -25,6 +25,7 @@ use SilverStripe\ORM\DataObject;
 
 class MyDataObject extends DataObject
 {
+    // ...
```

Comment:
Adding these where appropriate in this file to indicate we're skipping some best practices for the sake of brevity - we tend to do this with new code blocks, so this just gives consistency across this file.